app-emacs/projectile/projectile-2.8.0-r1.ebuild

```
@@ -25,3 +25,12 @@ src_test() {
 
 	elisp-test
 }
+
+pkg_postinst() {
+	#Descriptions for this packages' purpose were taken from Projectile's
+	#home page https://docs.projectile.mx/projectile/usage.html
+	optfeature_header "Performance can be gained in ${PN} by installing"
+	optfeature_header "the following packages:"
```

`optfeature_header` doesn't print anything by itself, but only sets up the header to be printed by the following `optfeature` commands. So when it is called twice consecutively, the first call will be ignored.
```suggestion
	optfeature_header "You can improve performance with the following packages:"
```

---

Hi @ulm, I was not aware of optfeature_header functionality, good to know! 
Thank you very much for the feedback. 
I submitted a new commit with your suggestion